### DISCUSSION TITLE
Embedding custom charts

### DISCUSSION THREAD

**Discussion by westandskif** [squidfunk/mkdocs-material#5580] [Q&A] [answered]
Hello! Can you please advise on the right approach to embed custom charts into docs? I'm working on my own charts library and want to document them using mkdocs-material.

**Comment by squidfunk** [comment, marked as answer]
Do it exactly as you would do it in plain HTML – you can use HTML in Markdown.

**Reply by westandskif** [reply]
Thank you very much!
